modules/ingest-common/src/yamlRestTest/resources/rest-api-spec/test/ingest/310_reroute_processor.yml

```diff
@@ -121,7 +121,7 @@ teardown:
   - do:
       index:
         refresh: true
-        index: logs-nginx-default
+        index: logs-router-default
```

Comment:
Based on the test name and the fact that the `logs-router` pipeline is unused, I suspect this is the intended index name.